public/templates/finesse_match.html

```diff
@@ -47,7 +47,7 @@
 
     <!-- import the webpage's javascript file -->
     <script src="https://code.jquery.com/jquery-3.3.1.js"></script>
-    <script src="/scripts/patientSignUp.js" defer></script>
+    <script src="/public/components/patientSignUp.js" defer></script>
```

Comment:
Should this go into `/public/assets/js/` it's ordinary JS as opposed to js that interacts with a system component like firebase services.

I think my original intent behind `/public/components/` was to store stuff that is either a self-contained subsystem or code that interacts with dependencies in some way.

Reply:
Could we keep them here for the time being as we work through the JS code for firebase? I suspect we might need a bit more discussion around this

Reply:
I think that's fine.